static/app/views/replays/replayTable/index.tsx

```diff
@@ -129,12 +152,16 @@ function ReplayTable({fetchError, isFetching, replays, sort, visibleColumns}: Pr
 }
 
 const StyledPanelTable = styled(PanelTable)<{
-  visibleColumns: Array<keyof typeof ReplayColumns>;
+  visibleColumns: ReplayColumns[];
 }>`
   grid-template-columns: ${p =>
     p.visibleColumns
       .filter(Boolean)
-      .map(column => (column === 'replay' ? 'minmax(100px, 1fr)' : 'max-content'))
+      .map(column =>
+        [ReplayColumns.replay, ReplayColumns.replay_slim].includes(column)
+          ? 'minmax(100px, 1fr)'
+          : 'max-content'
+      )
```

Comment:
This is prob ok because PanelTable won't be re-rendering too much but we should avoid dynamic styles for components that do (due to perf issues).